modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/indexer/IndexerWriterImpl.java

```diff
@@ -180,6 +180,10 @@ public void reindex(String[] ids) {
 
 				batchIndexingActionable.setCompanyId(companyId);
 
+				batchIndexingActionable.setInterval(
+					_batchIndexingHelper.getBulkSize(
+						_modelSearchSettings.getClassName()));
+
```

Comment:
hey @mbowerman, just curious if you considered putting this call in the `getBatchIndexingActionable()` method instead? off the top of my head that seems like it might be the better place to put it?